internal/cmd/run.go

```diff
@@ -125,8 +149,12 @@ func runCmd() *cobra.Command {
 					}
 
 					scriptRunText := "Running task"
+					if (runAll || category != "") && parallel {
+						scriptRunText = "Running"
+						blockNames = []string{blockColor.Sprint("all tasks")}
```

Comment:
Might be a good idea to specify the category here. So for example like "cat1 tasks" for category cat1, instead of "all tasks"